rke/templates/calico_v3.19.go

```diff
@@ -51,7 +52,13 @@ data:
           "nodename": "__KUBERNETES_NODE_NAME__",
           "mtu": __CNI_MTU__,
           "ipam": {
+{{- if eq (len $cidrs) 2 }}
```

Comment:
nit: could we also add `Rancher specific` comment here for later rebases?

Reply:
Added above as its an in-line json file